src/main/java/xyz/nucleoid/plasmid/game/common/GlobalWidgets.java

```diff
@@ -56,6 +58,74 @@ public SidebarWidget addSidebar(Text title) {
         return this.addWidget(new SidebarWidget(title));
     }
 
+    /**
+     * Adds a sidebar for all players associated with this {@link GlobalWidgets} instance.
+     *
+     * @return the created {@link SidebarWidget}
+     */
+    public SidebarWidget addSidebar() {
+        return this.addWidget(new SidebarWidget());
+    }
+
+    /**
+     * Adds a sidebar for selected players associated with this {@link GlobalWidgets} instance.
+     *
+     * @param playerChecker function returning true for players that can see this sidebar
+     * @return the created {@link SidebarWidget}
+     */
+    public SidebarWidget addSidebar(Text title, Function<ServerPlayerEntity, Boolean> playerChecker) {
```

Comment:
I think these would make sense as `Predicate<ServerPlayerEntity>`?

Reply:
Oh right, fixing it rn